sos/plugins/crash_report.py

```diff
@@ -0,0 +1,249 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+import os
+import sys
+import time
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+from sos.utilities import sos_get_command_output, find
+from datetime import datetime
+
+
+class CrashLog(Plugin):
+    """Crash kernel report
+    """
+    plugin_name = "crash_report"
+    profiles = ('system', 'debug')
+
+    dbg_vmlinux_path = '/usr/lib/debug/'
+    vmcore_stnd_path = '/var/crash'
+
+    dump_file_prefix = ''
+    crashed_kernel_name = ''
+    vmlinux_path = ''
+    message_to_log = ''
+    vmcore_file_path = ''
+    crash_script = ""
+    # crash result output file
+    crashed_output_file = '/var/log/os_crash_report.txt'
+
+    # list of crash command
+    crash_cmd_list = [
+                    ("sys", "\"SYSTEM INFORMATION:\""),
+                    ("mach", "\"MACHINE SPECIFIC DATA:\""),
+                    ("mach", "\"MACHINE SPECIFIC DATA:\""),
```

Comment:
I mean that `mach` is literally called twice with the same arguments:
```
                    ("mach", "\"MACHINE SPECIFIC DATA:\""),
                    ("mach", "\"MACHINE SPECIFIC DATA:\""),
```
I know `mach` provides different data based on flags but this looks like it just a duplicated line, since we call `mach -o` later, no?
```
                    ("mach -o", "\"OPALMSG LOG:\""),
```
Also, pretty sure everything that is output in `mach` without arguments is also in `help -m`, which also includes stack locations and various settings, but we don't have to include it if it's not needed.